### ISSUE TITLE
Undersetting mathml token regression from v2 to v3

### ISSUE THREAD

**Issue by dainiak** [mathjax/MathJax#2415]
### Issue Summary

This example [taken](https://math.meta.stackexchange.com/questions/23244/how-to-do-a-underbracket-with-mathjax) from StackExchange works perfectly in MathJax 2.7, but not in MathJax 3.0.5:
```\mathop{\underset{\mmlToken{mo}{⎵}}{78\,255\,300,00}}\limits_{10\text{ digits}}```

MJv2:
![image](https://user-images.githubusercontent.com/2994860/79671769-b9983a80-81d5-11ea-8ff7-62f139f576b4.png)

MJv3:
![image](https://user-images.githubusercontent.com/2994860/79671774-c74dc000-81d5-11ea-8cdb-061d7c7c7f4e.png)

### Technical details:

* MathJax v3.0.5 vs. v2.7
* Client OS: Windows 10
* Browser: Chrome 81

**Comment by dpvc** [comment]
Yes, there are some issues with stretchy characters that ned to be addressed.  In v2, there were options in the stretchy character day to allow pieces to be repositioned, but that was not carried over to v3.  We are doing work to extend the v3 font support this summer, and we hope to address such issues then.

**Comment by dpvc** [comment]
For the time being, if you are using CommonHTML output, you could add

``` html
<style>
mjx-stretchy-h.mjx-c23B5 > mjx-ext > mjx-c::before {
  vertical-align: -.24em;
  margin-bottom: -.24em;
}
</style>
```

to the page, which will fix this particular stretchy character.  Similar adjustments could be made for others that need it.

**Comment by dainiak** [comment]
Thank you! Is there a similar workaround for SVG? I only use SVG output module currently.

**Comment by dpvc** [comment]
Try

```
g[data-mml-node="mo"]:last-child >
  use[xlink|href$="2518"] + svg >
  use[xlink|href$="2212"] {
    transform: translateY(-250px) scaleX(100) translateX(-100px);
}
```

and see if that does the trick.

**Comment by dpvc** [comment]
OOPS, I left out the important

``` css
@namespace xlink "http://www.w3.org/1999/xlink";
```

line in my comment above.  I've added it in.  Note that it has to come above any other CSS declarations.

**Comment by dainiak** [comment]
Thank you, I’ll try that!

**Comment by dpvc** [comment]
The new fonts in v4.0 have the needed characters to produce the under bracket properly.
